core/src/main/scala/kanaloa/reactive/dispatcher/PerformanceSampler.scala

```diff
@@ -60,22 +60,21 @@ private[dispatcher] trait PerformanceSampler extends Actor {
       subscribers -= s
   }
 
-  def publishUtilization(idle: Int, poolSizeO: Option[Int]): Unit = {
-    poolSizeO.foreach { poolSize ⇒
-      val utilization = poolSize - idle
-      publish(PartialUtilization(utilization))
-      report(PoolUtilized(utilization))
-    }
+  def publishUtilization(idle: Int, poolSize: Int): Unit = {
+    val utilization = poolSize - idle
+    publish(PartialUtilization(utilization))
+    report(PoolUtilized(utilization))
   }
 
   def reportQueueLength(queueLength: QueueLength): Unit =
     report(WorkQueueLength(queueLength.value))
 
   def fullyUtilized(s: QueueStatus): Receive = handleSubscriptions orElse {
-    case DispatchResult(idle, workLeft) ⇒
+    case DispatchResult(idle, workLeft, full) ⇒
```

Comment:
um..idk..something that is more indicative of what it means.  I had to go back to the case class definition to figure out what that value was for, so maybe since it is a boolean, `isFullyUtilized`?